NuRadioReco/detector/response.py

```diff
@@ -304,24 +328,32 @@ def plot(self, ax1=None, show=False, in_dB=True, plt_kwargs={}):
         else:
             ax = ax1
 
-        for gain, weight, name in zip(self.__gains, self.__weights, self.__names):
+        for gain, weight, name, td in zip(self.__gains, self.__weights, self.__names, self.__time_delays):
             _gain = gain(freqs)
 
             name = name.replace("_", " ")
             ls = "-" if weight == 1 else "--"
 
+            if name.startswith("golden"):
+                name = name.replace("golden downhole components", "ref. comp.")
+
+            if name.endswith(" "):
+                name = name[:-1]
```

Comment:
`name.strip()` removes leading and trailing whitespace and saves one line over your implementation ;)

Reply:
Well if I fix it now you have to approve again

Reply:
To the extent that it is within my powers I give you permission to treat the first approval as the start of the 24h window. But I'm also perfectly happy with this staying like this.

Reply:
It does not remove whitespaces within the string?

Reply:
It removes leading and trailing whitespace only; alternatively there's `lstrip`/`rstrip` to do only one of those.

Reply:
crazy, not what I would have expected. I will fix it next time I touch that class